clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -2866,7 +2870,12 @@ private static void printErr(String errorMessage, Map<String, String> customMess
   }
 
   private static PubSubConsumerAdapter getConsumer(Properties consumerProps) {
+    PubSubMessageDeserializer pubSubMessageDeserializer = new PubSubMessageDeserializer(
+        new OptimizedKafkaValueSerializer(),
+        new LandFillObjectPool<>(KafkaMessageEnvelope::new),
+        new LandFillObjectPool<>(KafkaMessageEnvelope::new));
     return new ApacheKafkaConsumerAdapterFactory()
-        .create(new VeniceProperties(consumerProps), false, "admin-tool-topic-dumper");
+        .create(new VeniceProperties(consumerProps), false, pubSubMessageDeserializer, "");
```

Comment:
I think keeping the consumer name is better. :-)

Reply:
Done